requirements/docs-requirements.in

```diff
@@ -7,5 +7,6 @@ sphinxcontrib-django
 sphinx-rtd-theme
 
 pyOpenSSL # CI builds fail without OpenSSL
+msal # CI builds fail without OpenSSL
```

Comment:
I think this should be added in base-requirements.in and not here. This is not a docs requirement.

Alternately move the import in models.py into `get_all_members_of_the_idp` so it does not need to be added here.

Reply:
Thank you Daniel! I go with your second option : 11b11607f135468f8e9c84e184969e073dcd9e09